core/parser.mly

```diff
@@ -436,8 +423,8 @@ fun_kind:
 tlfunbinding:
 | fun_kind VARIABLE arg_lists perhaps_location block           { ($1, $2, $3, $4, $5)                }
 | OP pattern sigop pattern perhaps_location block              { ((dl_unl, false), WithPos.node $3, [[$2; $4]], $5, $6) }
-| OP PREFIXOP pattern perhaps_location block                   { ((dl_unl, false), $2, [[$3]], $4, $5)          }
-| OP pattern POSTFIXOP perhaps_location block                  { ((dl_unl, false), $3, [[$2]], $4, $5)          }
+| OP OPERATOR pattern perhaps_location block                   { ((dl_unl, false), $2, [[$3]], $4, $5)          }
+| OP pattern OPERATOR perhaps_location block                  { ((dl_unl, false), $3, [[$2]], $4, $5)          }
```

Comment:
```suggestion
| OP pattern OPERATOR perhaps_location block                   { ((dl_unl, false), $3, [[$2]], $4, $5)          }
```

(nit: alignment)

Reply:
There is a note to be made here, that I didn't remark in the source code. With this change the source language becomes more "permissive" in some sense. Previously it would be a syntax error to define a prefix operator using the postfix operator definition syntax. For example, the following would be a syntax error
```
prefix ++;
op v ++ { v + 1 }
```
with this patch it is permitted, because we cannot tell a priori whether `++` is a prefix, postfix, or infix operator.

Personally, I think we shouldn't spend any effort on rejecting the above example, it is just bad code practice.